src/TicketsSdkModal.tsx

```diff
@@ -22,7 +22,7 @@ export const TicketsSdkModal = ({
   return (
     <>
       {showTicketsModal && (
-        <View>
+        <View testID="ticketsSDKWrapper">
```

Comment:
should this be here?

Reply:
Ideally, I should be able to mock `requireNativeComponent` from `react-native` to return me a fake component so that I can check if it's rendered. I've tried it but when mocking `react-native` to overwrite `requireNativeComponent` I got plenty of issues. So in order to avoid them and still be able to test the showing/hiding, I gave the `testID`.

What I've tried is more or less this: 

```
jest.mock('react-native', () => {
  const ReactNative = jest.requireActual('react-native');
  const { View, Text } = ReactNative;
  return {
    ...ReactNative,
    requireNativeComponent: jest.fn(() => () => <View><Text>Hello fake</Text></View>),
  };
});
```